app/views/users/show.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="well well-sm">
+  <div>
+  <h5><%= current_user.email %></h5>
+  </div>
+</div>
+
+<p>
+<div class="col-md-8">
+  <%= form_for [@user, @item] do |f| %>
+    <%= f.label :name %>
+    <%= f.text_field :name, placeholder: "New List"  %>
+    <%= f.submit "Save" %>
+  <% end %>
+    <div class="form-group" >
+    </div>
+  </div>
+</p>
+
+<div class="container">
+  <div class="col-md-8">
+    <table>
+      <% @items.each do |i| %>
+        <tr>
+          <td><%= @item.name %></td>
+          <td><%= link_to @items_path %></td>
```

Comment:
This should just be `<%= link_to [@item.user, @item] %>` I think. `@items_path` refers to an instance variable -- since you haven't set an instance variable named `@items_path` it is nil, which is why you're getting an error.